core/install.sh

```diff
@@ -179,6 +179,9 @@ EOF
 echo "Install Traefik:"
 add-module traefik 1
 
+echo "Setting default admin password:"
+add-user --role owner --password Nethesis,1234 admin
```

Comment:
What about adding an hook to override it?

```suggestion
add-user --role owner --password "${ADMIN_PASSWORD:-Nethesis,1234}" admin
```

Reply:
I like it